101-anf-smb-volume/prereqs/DSC/CreateADPDC.ps1

```diff
@@ -0,0 +1,105 @@
+﻿configuration CreateADPDC 
```

Comment:
We should not provide the AD configuration part, this needs to focus on ANF only, there is already a dedicated quickstart template for that and we are referencing it on the description as pre-requisite. This comment applies to all resources related to bringing up a domain controller in this sample.

Reply:
this was part of the prerequisites. so I just copied all the files as is.

Reply:
Changed into a module